iOS/Projects/Features/Record/Sources/Domain/Entities/WorkoutMode.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  WorkoutMode.swift
+//  RecordFeature
+//
+//  Created by 안종표 on 2023/11/21.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - WorkoutMode
+
+enum WorkoutMode {
+  case run
+  case swim
+  case cycle
+}
+
+extension WorkoutMode {
+  var decription: String {
+    switch self {
+    case .run:
+      return "달리기"
+    case .swim:
+      return "수영"
+    case .cycle:
+      return "자전거"
+    }
+  }
+}
```

Comment:
p3: 이왕 description 넣은 거 CustomStringConvertible도 준수하는 건 어떠신가요? 나중에 Log 찍을 때 편할 것 같습니다 ㅎㅎ
```suggestion
extension WorkoutMode: CustomStringConvertible {
  var decription: String {
    switch self {
    case .run:
      return "달리기"
    case .swim:
      return "수영"
    case .cycle:
      return "자전거"
    }
  }
}
```

Reply:
바로 적용하겠습니다 ㅎㅎ